### PR TITLE
Prepares version update 1.1.1

### PR DESCRIPTION
**Pull request by vegaro**
CHANGELOG:
- Adds this library to Maven
    https://github.com/RevenueCat/purchases-hybrid-common/pull/31
- Fixes discountWithIdentifier always returns null
    https://github.com/RevenueCat/purchases-hybrid-common/pull/35
- Adds configure function to Android that lets pass a PlatformInfo
    https://github.com/RevenueCat/purchases-hybrid-common/pull/32
- Adds method to pass in a suite name to revenuecat
    https://github.com/RevenueCat/purchases-hybrid-common/pull/33

Also updates RELEASING.md

### CODE REVIEW

File: RELEASING.md [hunk 1]
```diff
@@ -1,10 +1,16 @@
...
 - Update `CHANGELOG.md`
 - Open a PR, merge and tag master.
-- Compile android aar: `cd android`, `./gradlew assembleRelease`. The .aar output will be in `common/build/outputs/aar/common-release.aar`
-- Create a github release, upload .aar to the release
+- Create a github release
+- Release Android:
+  1. Visit [Sonatype Nexus](https://oss.sonatype.org/)
+  1. Click on Staging Repositories on the left side
+  1. Scroll down to find the purchase repository
+  1. Select and click "Close" from the top menu. Why is it called close?
```
Comment (aboedo) [inline RELEASING.md:14]: LOL

File: RELEASING.md [hunk 2]
```diff
@@ -1,10 +1,16 @@
...
+  1. Once close is complete, repeat but this time selecting "Release"
 - run `pod trunk push PurchasesHybridCommon.podspec`
```
Comment (aboedo) [inline RELEASING.md:16]: can't wait until we automate this whole thing

**Comment by aboedo** [comment]
could you add https://github.com/RevenueCat/purchases-hybrid-common/pull/33 to the changelog?

**Comment by vegaro** [comment]
I am getting some warnings when releasing this in iOS:

```
 -> PurchasesHybridCommon (1.1.1)
    - NOTE  | xcodebuild:  note: Using new build system
    - NOTE  | xcodebuild:  note: Building targets in parallel
    - NOTE  | xcodebuild:  note: Planning build
    - NOTE  | xcodebuild:  note: Constructing build description
    - NOTE  | xcodebuild:  warning: Skipping code signing because the target does not have an Info.plist file and one is not being generated automatically. (in target 'App' from project 'App')
    - NOTE  | xcodebuild:  note: Execution policy exception registration failed and was skipped: Error Domain=NSPOSIXErrorDomain Code=1 "Operation not permitted" (in target 'Purchases' from project 'Pods')
    - WARN  | xcodebuild:  PurchasesHybridCommon/ios/PurchasesHybridCommon/PurchasesHybridCommon/RCPurchases+HybridAdditions.m:13:17: warning: method definition for '_setPushTokenString:' not found [-Wincomplete-implementation]
    - NOTE  | xcodebuild:  PurchasesHybridCommon/ios/PurchasesHybridCommon/PurchasesHybridCommon/RCPurchases+HybridAdditions.h:14:1: note: method '_setPushTokenString:' declared here
    - WARN  | xcodebuild:  PurchasesHybridCommon/ios/PurchasesHybridCommon/PurchasesHybridCommon/RCPurchases+HybridAdditions.m:13:17: warning: method definition for 'configureWithAPIKey:appUserID:observerMode:userDefaults:platformFlavor:platformFlavorVersion:' not found [-Wincomplete-implementation]
    - NOTE  | xcodebuild:  PurchasesHybridCommon/ios/PurchasesHybridCommon/PurchasesHybridCommon/RCPurchases+HybridAdditions.h:23:1: note: method 'configureWithAPIKey:appUserID:observerMode:userDefaults:platformFlavor:platformFlavorVersion:' declared here
    - NOTE  | xcodebuild:  note: Execution policy exception registration failed and was skipped: Error Domain=NSPOSIXErrorDomain Code=1 "Operation not permitted" (in target 'PurchasesHybridCommon' from project 'Pods')
    - NOTE  | xcodebuild:  note: Execution policy exception registration failed and was skipped: Error Domain=NSPOSIXErrorDomain Code=1 "Operation not permitted" (in target 'Pods-App' from project 'Pods')
    - NOTE  | xcodebuild:  note: Execution policy exception registration failed and was skipped: Error Domain=NSPOSIXErrorDomain Code=1 "Operation not permitted" (in target 'App' from project 'App')

[!] The spec did not pass validation, due to 2 warnings (but you can use `--allow-warnings` to ignore them).
```

It's safe to ignore right?

**Comment by vegaro** [comment]
Maven is timing out and giving 500 errors lol

**Comment by aboedo** [comment]
we get a similar thing for purchases-ios. I haven't looked much into it but I think it should be fixable - we probably just need to add the path to our info.plist to the podspec

**Pull request merged**